buffer/src/test/java/io/netty/buffer/PooledBigEndianDirectByteBufTest.java

```diff
@@ -22,20 +22,12 @@
 /**
  * Tests big-endian direct channel buffers
  */
-public class PooledBigEndianDirectByteBufTest extends AbstractByteBufTest {
-
-    private ByteBuf buffer;
+public class PooledBigEndianDirectByteBufTest extends AbstractPooledByteBufTest {
 
     @Override
-    protected ByteBuf newBuffer(int length) {
-        buffer = PooledByteBufAllocator.DEFAULT.directBuffer(length);
+    protected ByteBuf alloc(int length) {
```

Comment:
we still have `newBuffer(...)`., `alloc(...)` was added for the actually allocation of the buffer. So both are there now.